iree/vm/test/arithmetic_ops_i64.mlir

```diff
@@ -1,10 +1,22 @@
 vm.module @arithmetic_ops_i64 {
 
   //===--------------------------------------------------------------------===//
-  // I64 Arithmetic
+  // ExtI64: Native integer arithmetic
   //===--------------------------------------------------------------------===//
 
+  // TODO: The CModuleTarget enforces exports to be ordered.
   vm.export @test_add_i64
+  vm.export @test_and_i64
+  vm.export @test_div_i64s
+  vm.export @test_div_i64u
+  vm.export @test_mul_i64
+  vm.export @test_not_i64
+  vm.export @test_or_i64
+  vm.export @test_rem_i64s
+  vm.export @test_rem_i64u
+  vm.export @test_sub_i64
+  vm.export @test_xor_i64
+
   vm.func @test_add_i64() {
     %c1 = vm.const.i64 1 : i64
```

Comment:
How about using values, which would overflow for 32bit integers?

Reply:
Yeah, I had this in mind as well... But, hm, `test_add_i64` was already present ;)